From 1506ecdfd0f91d4b8b39c01bad1fe301193d6e82 Mon Sep 17 00:00:00 2001
From: David Hallas <david@davidhallas.dk>
Date: Sun, 11 Nov 2018 10:26:31 -0500
Subject: Add StartupWMClass to .desktop file

The org.kde.konsole.desktop file is missing a StartupWMClass setting.

BUG: 372441
FIXED-IN: 18.12

Differential Revision: https://phabricator.kde.org/D16546
---
 desktop/org.kde.konsole.desktop | 1 +
 1 file changed, 1 insertion(+)

diff --git a/desktop/org.kde.konsole.desktop b/desktop/org.kde.konsole.desktop
index adc472c..86c2160 100755
--- a/desktop/org.kde.konsole.desktop
+++ b/desktop/org.kde.konsole.desktop
@@ -10,6 +10,7 @@ X-DocPath=konsole/index.html
 X-DBUS-StartupType=Unique
 StartupNotify=true
 X-KDE-AuthorizeAction=shell_access
+StartupWMClass=konsole
 
 Name=Konsole
 Name[af]=Konsole
-- 
cgit v0.11.2
